recipes/fast-cdr/all/conanfile.py

```diff
@@ -59,23 +59,12 @@ def _source_subfolder(self):
 
     def _create_cmake_module_alias_target(self):
```

Comment:
Typo ( I couldn't find the regular implementation when looking)

```suggestion
    def _create_cmake_module_alias_targets(self):
```